<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lbphMethod</class>
 <widget class="QWidget" name="lbphMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1010</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>970</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>вернуться</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbph_test_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тестовое преобразованное</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тестовое изображение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ LBPH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тестовое преобразованное</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbph_test">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Тестовое</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevImageButton">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>910</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbph_answer_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ преобразованное</string>
   </property>
  </widget>
  <widget class="QWidget" name="histogramWidget" native="true">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>1231</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbph_answer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextImageButton">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>850</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>далее</string>
   </property>
  </widget>
  <widget class="QWidget" name="histogramWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>1231</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ преобразованный</string>
   </property>
  </widget>
  <widget class="QLabel" name="distText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Разница между тестом и эталоном</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>750</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="distText_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>870</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Средняя разница между тестами и эталонами</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>930</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>750</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="distText_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>690</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Кол-во ошибок метода</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
